### ISSUE TITLE
Ensure master builds do not have useless vendored glimmer/di code

### ISSUE THREAD

**Issue by mixonic** [emberjs/ember.js#15024]
Landing https://github.com/emberjs/ember.js/pull/14896 was probably over eager as the vendored dependency may be included in builds. We should not include it in builds unless it is being used (likely toggled by a feature flag).

**Comment by chancancode** [comment]
This will affect the 2.13 beta builds

**Comment by cibernox** [comment]
I don't know if it's related, but i can't update to 2.13-beta and the error message seems related.

I have ~50 tests failing with 

```
not ok 1 Chrome 57.0 - Global error: Uncaught Error: Could not find module glimmer-runtime at http://localhost:7357/assets/vendor.js, line 15832
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: Could not find module glimmer-runtime at http://localhost:7357/assets/vendor.js, line 15832\n' }
```

Other fail with 
```
not ok 469 Chrome 57.0 - TestLoader Failures: tutoring-frontend/tests/integration/components/tutor-column/component-test: could not be loaded
    ---
        actual: >
            null
        expected: >
            null
        stack: >
            Error: you must set a resolver with `testResolver.set(resolver)`
```

The app was in ember-cli 2.13-beta already, so the diff of the commit to cause this error is just:

```diff
diff --git a/package.json b/package.json
index a5a774e..d9de739 100644
--- a/package.json
+++ b/package.json
@@ -78,7 +78,7 @@
     "ember-responsive": "^2.0.2",
     "ember-route-action-helper": "^2.0.2",
     "ember-simple-auth": "1.1.0",
-    "ember-source": "~2.12.0",
+    "ember-source": "~2.13.0-beta.0",
     "ember-test-selectors": "^0.2.1",
     "ember-tether": "0.4.1",
     "ember-tooltips": "^2.8.0",
```

**Comment by mixonic** [comment]
The dependency should be removed as of https://github.com/emberjs/ember.js/pull/15044

@cibernox is this persisting for you on 2.13? Seems like a massive break in the beta if you're getting that error? Perhaps you can open a reproduction?

**Comment by mixonic** [comment]
I'm going to close this issue- @cibernox if you're still running into something please do open a fresh issue.

**Comment by rwjblue** [comment]
@mixonic - I still need to fix this for 2.13 (current beta branch)
